std_msgs/Empty empty
---
sensor_msgs/CameraInfo camera_info
